src/components/TransactionTable.vue

```diff
@@ -72,6 +72,7 @@ export default {
         return {
             rows: [],
             columns,
+            filterUpdated: false,
```

Comment:
since `filterUpdated` isnt referenced anywhere else, i think it can be removed? right now it is being set in the filter handler but not used anywhere

Reply:
it's used in the handler, it prevents an unnecessary first call of the handler when page loads.
it's set to true after that first load and then it did its job.
There could be a better way to do that though, I just didn't find it.

Reply:
No you're right, I missed the return after it is set. Makes sense, sorry about that